src/rust/engine/process_execution/src/local.rs

```diff
@@ -114,6 +115,79 @@ impl CommandRunner {
   }
 }
 
+lazy_static! {
+  static ref IS_LIKELY_IN_DOCKER: bool = is_likely_in_docker().unwrap_or_else(|e| {
+    log::info!("Failed to detect whether we are running in docker: {}", e);
```

Comment:
Will this be logged every time Pants runs? This might be better as a debug log. There's no remedy for the user in this situation, so I could imagine it being a frustrating message to get every time.

Reply:
@Eric-Arellano : I would be genuinely curious to get a bug report if it failed... would it be helpful to include more information here about how to file?

Reply:
Yes, for sure! Big thing I think we should aim for with every error message is that there's a remedy the user can take.